docs/sql-ref-syntax-aux-describe-database.md

```diff
@@ -18,5 +18,23 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+​
+Return the metadata of an existing database(name, comment and location). If the database does not exist,
+an exception is thrown.
+
+When `extended` is specified, it also shows the database's properties.
+ 
+You can use the abbreviation `DESC` for the `DESCRIBE` statement.
+
+### Syntax
+{% highlight sql %}
+[DESC | DESCRIBE] DATABASE [EXTENDED] db_name 
+{% endhighlight %}
+
+### Example
+{% highlight sql %}
+DESCRIBE DATABASE EXTENDED tempdb
```

Comment:
Can you provide any output? this doc page isn't giving much additional info (though there is not a lot to know, granted)

Reply:
sure